package-res/resources/web/pentaho/type/changes/Update.js

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define([
+  "./Operation"
+], function(Operation) {
+  "use strict";
+  
+  /**
+   * @name Update
+   * @memberOf pentaho.type.changes
+   * @class
+   * @extends pentaho.type.changes.Operation
+   * @amd pentaho/type/changes/Update
+   *
+   * @classDesc Describes an operation that updates an element in a list.
+   * In an `update` operation, the reference to the element does not change, but its contents does.
```

Comment:
suggesting wording:

In an `update` operation, the reference to the element does not change, but its content does.